pkg/virt-controller/services/rendervolumes.go

```diff
@@ -839,3 +872,22 @@ func (vr *VolumeRenderer) handleDownwardMetrics(volume v1.Volume) {
 		MountPath: config.DownwardMetricDisksDir,
 	})
 }
+
+// shouldAddLauncherBinaryVolume decides if we need to add the launcher image volume.
+// Even if only one init container is required for digest extraction, we must add the
+// volume that contains the binary used by that init container. Without this volume,
+// the init container cannot run.
```

Comment:
I'm not sure I understand this TBH. What's the purpose of adding another image volume with the virt-launcher image?

Reply:
Thanks for explaining!